services/myget/myget.tester.js

```diff
@@ -58,11 +58,12 @@ t.create('total downloads (unexpected second response)')
   .intercept(nock =>
     nock('https://api-v2v3search-0.nuget.org')
       .get(
-        '/query?q=packageid:mongodb.driver.core&prerelease=true&semVerLevel=2'
+        '/query?q=packageid%3Amongodb.driver.core&prerelease=true&semVerLevel=2'
       )
       .reply(invalidJSON)
   )
-  .expectJSON({ name: 'downloads', value: 'invalid' })
+  // .expectJSON({ name: 'myget', value: 'unparseable json response' })
+  .expectJSON({ name: 'downloads', value: 'unparseable json response' })
```

Comment:
I'm not sure I feel strongly about the label on some of these error badges. They're a bit inconsistent, some showing the service name and some showing "downloads."